packages/cli/src/databases/migrations/mysqldb/1681134145996-AddUserActivatedProperty.ts

```diff
@@ -0,0 +1,59 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { getTablePrefix, logMigrationEnd, logMigrationStart } from '@db/utils/migrationHelpers';
+import { userSettings } from '@/Interfaces';
```

Comment:
```suggestion
import type { userSettings } from '@/Interfaces';
```

Reply:
This should have been caught by `consistent-type-imports` but apparently linting is disabled for migrations.